components/datetime/src/raw/zoned_datetime.rs

```diff
@@ -119,7 +119,7 @@ impl ZonedDateTimeFormatter {
     where
         P: DataProvider<TimeSymbolsV1Marker>
             + DataProvider<TimeLengthsV1Marker>
-            + DataProvider<WeekDataV1Marker>
+            + DataProvider<WeekDataV2Marker>
```

Comment:
Discussion: Hmm, I don't _really_ want to add [Zoned]Date[Time]Formatter compatibility constructors at this time. I will be adding them later with the neo symbols. It's a fairly complex part of the code.

@robertbastian would you approve of keeping both v1 and v2 in baked data, using v1 on these old bounds, and then I can use v2 on the new DateTimeFormatter constructors I'm building? Then if everything goes smoothly maybe we can get rid of V1 baked data in 1.5.

Reply:
yeah I guess that's fine as well, it's not a lot of data